src/functions/verify/index.ts

```diff
@@ -0,0 +1,15 @@
+import type { AWS } from "@serverless/typescript";
+import { handlerPath } from "@libs/handlerResolver";
+
+export default <AWS["functions"][string]>{
+  handler: `${handlerPath(__dirname)}/handler.main`,
+  events: [
+    {
+      http: {
```

Comment:
is this how to configure serverless through ts? must be part of the function?

Reply:
Not too sure if it's a must. This was provided in the Serverless TypeScript template.